node-hub/dora-gr00t/tests/test_dora_gr00t.py

```diff
@@ -0,0 +1,18 @@
+"""TODO: Add docstring."""
+
+import os
+import sys
+
+import pytest
+
+# Add the parent directory to the path to make the package discoverable
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

Comment:
We don't want to import groot at runtime. We want it to be linked to the package.